docs/docs/introduction.md

```diff
@@ -4,23 +4,13 @@ sidebar_position: 0
 
 # Introduction
 
-This is a specification for the packaging, installation, update, and monitoring process of [Eigen Layer](https://www.eigenlayer.xyz/) Nodes. AVS is used to participate in the Eigen Layer restaking system, to know more about Eigen Layer see its [documentation page](https://docs.eigenlayer.xyz/overview/readme). 
+The [EigenLayer](https://www.eigenlayer.xyz/) ecosystem allows, among other incredible features, Ethereum validators to restake their ETH and provide services to the network, such as a Data Availability Layer with a particular type of node software called actively validated services (AVS). 
 
-The current version of the specification is `v0.0.1`.
+With the increasing number of AVS use cases, there is a need for a convention on how to develop, maintain, create interoperability and use them. Eigenlayer has partnered with Nethermind to create an AVS Node Specification with those goals in mind. We also decided to build an AVS Node Setup Wizard to install and manage AVS Nodes that follow such specification. This documentation describes the AVS Node Specification and the AVS Node Setup Wizard.
```

Comment:
Should we mention _monitoring_ in the first sentence?

Reply:
Done [b781a58](https://github.com/NethermindEth/el-node-spec/pull/54/commits/b781a58a87e5e3d868124c43497aa0122840fee6)